tests/errors/tests/errors.js

```diff
@@ -128,4 +128,19 @@ describe("errors", () => {
       assert.equal(err.code, 300 + 125);
     }
   });
+
+  it("Emits a account not initialized error", async () => {
+    try {
+      const tx = await program.rpc.accountNotInitializedError({
+        accounts: {
+          notInitializedAccount: (new anchor.web3.Keypair()).publicKey
+        },
+      });
+      assert.ok(false);
```

Comment:
https://nodejs.org/api/assert.html#assertfailmessage :)

Reply:
Just copy from this one:
https://github.com/project-serum/anchor/blob/2cbd973a6d2d2d4e5661f6a6d2cbdf2ab9a9e753/tests/errors/tests/errors.js#L123
Will fix to `assert.fail`